dace/transformation/dataflow/otf_map_fusion.py

```diff
@@ -175,12 +262,87 @@ def map_memlet(memlet):
         return new_inter_nodes
 
     @staticmethod
-    def _memlet_offsets(base_memlet, offset_memlet):
-        """Compute subset offset of `offset_memlet` relative to `base_memlet`."""
-        def offset(base_range, offset_range):
-            b0, e0, s0 = base_range
-            b1, e1, s1 = offset_range
-            assert e1 - e0 == b1 - b0 and s0 == s1
-            return int(e1 - e0)
-
-        return tuple(offset(b, o) for b, o in zip(base_memlet.subset.ranges, offset_memlet.subset.ranges))
+    def solve(first_params, write_accesses, second_params, read_accesses):
+        """ Infers the memory access for the write memlet given the
+            location/parameters of the read access.
+
+            Example:
+            - Write memlet: A[i + 1, j]
+            - Read memlet: A[k, l]
+            - Infer: k -> i - 1, l - > j
```

Comment:
improper format, sphinx will complain. Instead, put the expressions in double-backticks:
```
``A[i + 1, j]``
```